test/remote/gateways/remote_paybox_direct_3ds_test.rb

```diff
@@ -0,0 +1,141 @@
+# encoding: utf-8
+
+require 'test_helper'
+
+class RemotePayboxDirect3DSTest < Test::Unit::TestCase
+  def setup
+    fixtures = fixtures(:paybox_direct)
+    @gateway = PayboxDirectGateway.new(fixtures)
+
+    @amount = 100
+    @credit_card = credit_card(fixtures[:credit_card_ok_3ds])
+    @declined_card = credit_card(fixtures[:credit_card_nok_3ds])
+    @unenrolled_card = credit_card(fixtures[:credit_card_ok_3ds_not_enrolled])
+
+    @options = {
+      order_id: '1',
+      billing_address: address,
+      description: 'Store Purchase',
+      three_d_secure: true,
```

Comment:
This is wrong, the proper format according to [this documentation](https://github.com/activemerchant/active_merchant/wiki/Standardized-3DS-Fields) should be:
```ruby
 @options = {
      order_id: '1',
      billing_address: address,
      description: 'Store Purchase',
      three_d_secure: {
        eci: '02',
        cavv: 'jJ81HADVRtXfCBATEp01CJUAAAA=',
        xid: '00000000000000000501',
        cavv_algorithm: '1'
      }
}
```